hw-05/src/routes/users/createUser.js

```diff
@@ -0,0 +1,33 @@
+const User = require('./userSchema');
+const bcrypt = require('bcrypt');
+
+const createUser = async (request, response) => {
+  try {
+    const user = request.body;
+    const notUniqueUser = await User.findOne({
+      email: request.body.email
+    });
+    if (notUniqueUser) {
+      return response.status(400).send("User already registered.")
+    };
+    const hashedPassword = await bcrypt.hash(user.password, 10);

```

Comment:
10 it is magic number 

https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)